### PR TITLE
Added html form and change name account

### PR DESCRIPTION
**Pull request by metja**
[DEMO LINK](https://metja.github.io/layout_html-form/)
- [TEST REPORT LINK](https://metja.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,175 @@
...
+        <legend>Personal information</legend>
+
+        <label>
+          Surname:
+          <input
+          name="surname"
+          type="text"
+          autocomplete="off"
+          required
+          >
```
Comment (liliia-livandovskaya) [inline src/index.html:26]:
```suggestion
          <input
            name="surname"
            type="text"
            autocomplete="off"
            required
          >
```

Comment (liliia-livandovskaya) [reply src/index.html:26]: checklist p.3 https://github.com/mate-academy/layout_html-form/blob/master/checklist.md
fix it all over the code

File: src/style.css [hunk 2]
```diff
@@ -1 +1,10 @@
 /* styles go here */
```
Comment (liliia-livandovskaya) [inline src/style.css:1]: remove comments

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
GJ! Deploy the project and pay attention to the comments 
Screenshot 2022-10-26 at 16 41 28

**Comment by metja** [comment]
I replaced the indents in html, but my deploy does not work correctly

ср, 26 жовт. 2022 р. о 16:44 Liliia Livandovskaya ***@***.***>
пише:

**Pull request closed without merging**